src/screens/journeysDisplay.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+//import React from 'react';
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { Text, Button, DataTable } from 'react-native-paper';
+import { ScreenNavigationProps } from '../routes';
+
+const styles = StyleSheet.create({
+  containerStyle: {
```

Comment:
I wouldn't keep "style" in these names, it makes them more verbose and I can already see they're styles because you'll access them everywhere by using `styles.spacer` etc